### DISCUSSION TITLE
Apple Music

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#6] [General]
# Apple Music

A community-driven list of apps and games that work, are emulated or don't work on Windows on ARM

https://windowsonarm.org/

**Comment by sundave1998** [comment]
I can run apple music via simulation on SP 11 w/ X Plus

**Reply by OpenSource03** [reply]
Hi, thanks for mentioning, we will retest it and update if necessary :)

**Reply by OpenSource03** [reply]
This has been corrected.
